bp/tests/Resource Files/fatal_tests_attempt_3.xml

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <testsuites name="Selected tests" tests="1" failures="0" errors="0">
     <testsuite tests="1" failures="0" errors="0" name="BPSampleAppFatalErrorTests.xctest">
+        <testsuite tests="0" failures="0" errors="0" name="BPSampleAppFatalErrorTests"></testsuite>
```

Comment:
This is a bit odd... do you know why it's printing `tests="0"` here?

Reply:
@ob  According to this simulator log, during the third attempt, Bluepill didn't know that the bundle `BPSampleAppFatalErrorTests` has no more tests left to execute. So, it kicked it off in a simulator and passed the test suite with zero tests executed.
```
XCTestOutputBarrierTest Suite 'BPSampleAppFatalErrorTests.xctest' started at 2021-09-16 14:11:32.980
XCTestOutputBarrierTest Suite 'BPSampleAppFatalErrorTests' started at 2021-09-16 14:11:32.981
XCTestOutputBarrierTest Suite 'BPSampleAppFatalErrorTests' passed at 2021-09-16 14:11:32.981.
	 Executed 0 tests, with 0 failures (0 unexpected) in 0.000 (0.000) seconds
```

Reply:
I am checking why **this started happening** in Xcode 13/iOS 15.0.

Reply:
Rebased the branch after https://github.com/MobileNativeFoundation/bluepill/pull/513 and reverted my fix to the test case temporarily.

Reply:
Put the fix back on. In iOS 15 Apple seems to have changed the way it handles test class with no tests to execute.